fabric-enchantments-v1/src/main/java/net/fabricmc/fabric/mixin/enchantment/MixinEnchantingTableContainer.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.mixin.enchantment;
+
+import net.fabricmc.fabric.api.enchantment.EnchantingPowerProvider;
+import net.minecraft.block.BlockState;
+import net.minecraft.container.EnchantingTableContainer;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.ModifyVariable;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+@Mixin(EnchantingTableContainer.class)
+public abstract class MixinEnchantingTableContainer {
+	private World fabric_world;
+	private BlockPos fabric_blockPos;
+
+	@Inject(method = "method_17411", at = @At("HEAD"))
```

Comment:
This should be mapped in Yarn before this PR is merged.

Reply:
It's a lambda ;)

Reply:
Would probably make a good comment then 😉